tests/unit/worker/test_task_executor.py

```diff
@@ -106,23 +104,6 @@ async def test_task_executor(random_task_class, persistent):
     assert document["description"] == "Execute random task"
 
 
-def test_task_has_been_implemented(random_task_class):
```

Comment:
this test shouldn't be needed anymore since you shouldn't be able to initialize a `BaseTask` on its own, and you should need to specify the `payload_class` when creating a new task or else the code will fail to build.